Element margins

nikku opened the pull request:
As discussed, this improves element margins in the form editor to better match the rendered form look. This is a quick fix. We'll likely ship further improvements later this quarter.

#### Viewer

![image](https://user-images.githubusercontent.com/58601/119120265-61f97d80-ba2c-11eb-840c-c671689ca6c9.png)

#### Editor (decreased margins)

![image](https://user-images.githubusercontent.com/58601/119120398-83f30000-ba2c-11eb-805d-dcbe67fee3af.png)

andreasgeier reviewed (approved):
As discussed, the UI will be revisited in this quarter, including the editor's spacing. 
But I appreciate this quick fix.

nikku commented:
@andreasgeier @philippfromme please have another look, as I've added a few more things onto the bucket:

* Adjust editor form field margins to fit viewer
* The form looks muted now, as specified. This was simply a bug in our CSS stylesheet https://github.com/bpmn-io/form-js/pull/63/commits/9eafb24ea08486030c6351e5c17efbdde8323ce3
* Added a text formating hint https://github.com/bpmn-io/form-js/pull/63/commits/1cfcbdb266ff174b5a61a2f40a27812992a0503d
* Decreased visual presence of descriptions https://github.com/bpmn-io/form-js/pull/63/commits/f113ada4ff60bcb9cba176b3120b226ad779ce5a

I've also updated the PR to indicate that this is a round of quick fixes, rather then the end of things :wink:.

philippfromme reviewed (approved):
👏🏻
